services/api/src/admin_new_rsu.py

```diff
@@ -193,18 +199,37 @@ def options(self):
         # CORS support
         return ("", 204, self.options_headers)
 
-    def get(self):
+    @require_permission(
+        required_role=ORG_ROLE_LITERAL.OPERATOR,
```

Comment:
question: for these admin interactions are we allowing operator to add/edit RSU? I thought that was an ADMIN operation?

Reply:
this get may be fine to allow OPERATOR to do, but what about the post below?

Reply:
Same as below - Operator should have access to editing devices, just not editing users